src/foam/dao/history/HistoryRecordServiceServer.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.dao.history;
```

Comment:
Also, why did you create this as java class and not modelled?

Reply:
It was going to be all java code so I went with java class instead of modelling it. Is there a benefit of modelling it in this case?

Reply:
Updated